### ISSUE TITLE
Multitenancy: Auditing tab is broken

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#2752]
Error:
```
Oct 31 19:12:06 management httpd_admin: httpd.admin(31648) ERROR: [mac:unknown] Database query failed with non retryable error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'AND `radius_audit_log`.`tenant_id` = '1' ) ) ORDER BY `id` DESC LIMIT 25 OFFSET ' at line 1 (errno: 1064) [SELECT `radius_audit_log`.`id`, `radius_audit_log`.`tenant_id`, `radius_audit_log`.`created_at`, `radius_audit_log`.`mac`, `radius_audit_log`.`ip`, `radius_audit_log`.`computer_name`, `radius_audit_log`.`user_name`, `radius_audit_log`.`stripped_user_name`, `radius_audit_log`.`realm`, `radius_audit_log`.`event_type`, `radius_audit_log`.`switch_id`, `radius_audit_log`.`switch_mac`, `radius_audit_log`.`switch_ip_address`, `radius_audit_log`.`radius_source_ip_address`, `radius_audit_log`.`called_station_id`, `radius_audit_log`.`calling_station_id`, `radius_audit_log`.`nas_port_type`, `radius_audit_log`.`ssid`, `radius_audit_log`.`nas_port_id`, `radius_audit_log`.`ifindex`, `radius_audit_log`.`nas_port`, `radius_audit_log`.`connection_type`, `radius_audit_log`.`nas_ip_address`, `radius_audit_log`.`nas_identifier`, `radius_audit_log`.`auth_status`, `radius_audit_log`.`reason`, `radius_audit_log`.`auth_type`, `radius_audit_log`.`eap_type`, `radius_audit_log`.`role`, `radius_audit_log`.`node_status`, `radius_audit_log`.`profile`, `radius_audit_log`.`source`, `radius_audit_log`.`auto_reg`, `radius_audit_log`.`is_phone`, `radius_audit_log`.`pf_domain`, `radius_audit_log`.`uuid`, `radius_audit_log`.`radius_request`, `radius_audit_log`.`radius_reply`, `radius_audit_log`.`request_time` FROM `radius_audit_log` WHERE ( (  AND `radius_audit_log`.`tenant_id` = ? ) ) ORDER BY `id` DESC LIMIT ? OFFSET ?]{1, 25, 0} (pf::dal::db_execute)
```

**Comment by jrouzierinverse** [comment]
Fixed by af2dd35375fc68380c6a76529dadfd6e6b4035d1

**Comment by extrafu** [comment]
Should be closed?
